examples/scrap_clipboard.py

```diff
@@ -55,9 +55,9 @@ def usage():
                 if r and len(r) > 500:
                     print("Type %s : (large %i byte buffer)" % (t, len(r)))
                 elif r is None:
-                    print("Type %s : None" % (t,))
+                    print(f"Type {t} : None")
                 else:
-                    print("Type %s : '%s'" % (t, r.decode("ascii", "ignore")))
+                    print("Type {} : '{}'".format(t, r.decode("ascii", "ignore")))
```

Comment:
```suggestion
                    print(f"Type {t} : '{r.decode("ascii", "ignore")}'")
```

Reply:
ok, just read your comment above about pyupgrade to try to not harm readability. Here the suggestion if probably as readable.

Reply:
The suggestion you put up would actually result in a syntax error... because of the nested `""` usage

Reply:
note to self: test your suggestions

Reply:
```suggestion
                    print("Type {t} : '{r.decode('ascii', 'ignore')}'")
```

This should work! (at least pycharm suggested that... ;) )